.github/workflows/qa.yml

```diff
@@ -1,26 +1,70 @@
 name: Q&A
 
 on:
-  pull_request:
+    pull_request:
 
 defaults:
-  run:
-    shell: bash
+    run:
+        shell: bash
 
 jobs:
 
-  tests:
-    name: Checks
-    runs-on: Ubuntu-20.04
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-
-      - name: No symlinks
-        run: |
-          SYMLINKS=$(find -type l)
-          if [[ "" != "$SYMLINKS" ]]; then
-            echo "::error::Symlinks are not allowed"
-            echo "Found $SYMLINKS"
-          fi
+    tests:
+        name: Checks
+        runs-on: Ubuntu-20.04
+
+        steps:
+            -
+                name: Checkout
+                uses: actions/checkout@v2
+
+            -
+                name: No symlinks
+                if: always()
```

Comment:
this should be `if: "always() and steps.end_of_setup.outcome == 'success'"` (with an id `end_of_setup` set on the `actions/checkout` step).
We don't want to run those linting steps if checking out the code failed, as that depends on it.

Reply:
updated